Update copy for Free and Enhanced Listings merge

eason9487 opened the pull request:
### Changes proposed in this Pull Request:

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

Closes #1296 

- Remove all "Partially Active" wordings or sentences.
- Remove "standard" or "enhanced" listings wordings.

### Screenshots:

#### 📷  Overview statistics

![2022-03-14 16 01 33](https://user-images.githubusercontent.com/17420811/158133645-aaee585e-b16b-4f94-8d5e-78f704b7a84e.png)

#### 📷  Popup of Overview statistics

![2022-03-14 16 00 07](https://user-images.githubusercontent.com/17420811/158133654-e1b02956-1e3a-4745-bff5-ae11c269760d.png)

#### 📷  Issues to resolve table

The new screenshot after adopting the suggestion in https://github.com/woocommerce/google-listings-and-ads/pull/1325#issuecomment-1135936945.

![2022-05-25 19 06 54](https://user-images.githubusercontent.com/17420811/170249302-ff404cbd-b74d-4f28-825a-520638c7ef71.png)

The ~new~ old screenshot after resolving code conflicts and rebasing in https://github.com/woocommerce/google-listings-and-ads/pull/1325#issuecomment-1135753523.
![2022-05-24 18 17 33](https://user-images.githubusercontent.com/17420811/170012562-7ce78f81-ec22-4ff2-8679-f5ffa3f0b3eb.png)

The original screenshot when opening PR.
![2022-03-14 16 03 09](https://user-images.githubusercontent.com/17420811/158133667-8881109e-e341-4b98-9af6-fb0a494ce157.png)

### Detailed test instructions:

1. Go to the Product Feed page.
2. Check if the copies are the same as new copies in the spreadsheet of pcTzPl-nO-p2.
3. Global search `partially`, `standard`, and, `enhanced` in the codebase to check if there are related copies that are still using these wordings.

### Changelog entry

> Tweak - Update copy for Free and Enhanced Listings merge.

eason9487 committed:
Update copy for free listings merge

eason9487 committed:
Adjust a code comment to describe how the Overview's Active is calculated

eason9487 commented on src/API/Site/Controllers/MerchantCenter/IssuesController.php:
```diff
@@ -239,10 +239,10 @@ protected function get_schema_title(): string {
 		$is_account_issue = MerchantStatuses::TYPE_ACCOUNT === $issue['type'];
-		if ( $is_account_issue && "Account isn't eligible for enhanced free listings" === $issue['issue'] ) {
-			$issue['issue']      = 'Show products on additional surfaces across Google through enhanced free listings';
+		if ( $is_account_issue && 'merchant_quality_low' === $issue['code'] ) {
```
Change to compare with the issue `code`. Ref: https://developers.google.com/shopping-content/guides/account-issues#merchant_quality_low

eason9487 committed:
Compatible with two possible account issues before/after Free and Enhanced Listings merge

ecgan commented on src/API/Site/Controllers/MerchantCenter/IssuesController.php:
```diff
@@ -239,10 +239,21 @@ protected function get_schema_title(): string {
+		 *       if its issue code is the same as 'merchant_quality_low'
+		 *       after Free and Enhanced Listings merge.
+		 */
+		if ( $is_account_issue && ( 'merchant_quality_low' === $issue['code'] || "Account isn't eligible for free listings" === $issue['issue'] ) ) {
```
❔ I've read the code comment, Google docs reference link and P2, but still wondering if we really need the `"Account isn't eligible for free listings" === $issue['issue']` part. I removed them manually in my testing and it still seems to work good. 

I'm thinking that we should have checked based on `'merchant_quality_low' === $issue['code']` in the first place, and checking based on `$issue['issue']` (whether it is "enhanced free listings" in the old code or "free listings" in the new code) isn't necessary. 

The TODO code comment says "Remove the condition of matching the $issue['issue'] if its issue code is the same as 'merchant_quality_low' after Free and Enhanced Listings merge." Will it be not same? I guess we don't know the answer and we are just trying to be a bit defensive here? If that's the case, I think that's fine.

eason9487 replied:
> Will it be not same? I guess we don't know the answer and we are just trying to be a bit defensive here? If that's the case, I think that's fine.

Yes. So far we only know that the content of `$issue['issue']` will become "Account isn't eligible for free listings", but it is not certain if they will use the same `$issue['code']`, so a compatible approach is taken here. After Google API response has been updated, the condition needs to be left here will be clear.

ecgan reviewed (approved):
Tested and compared the UI with the new copies in the spreadsheet, everything looks good. Left a minor ❔ comment below on the PHP conditional check on `$issue`. LGTM 👍

ecgan commented:
I have added `status: blocked` label in this PR, as we are pending confirmation from Google side.

eason9487 commented:
📝  Related PR #1437

puntope commented:
> 📝 Related PR https://github.com/woocommerce/google-listings-and-ads/pull/1437

Yeah, I wanted to move maybe_override_issue to before database insertion instead to avoid inconsistencies when hiding warning issues. 

But it can be adjusted easily after merge :)

eason9487 commented:
Since there are code conflicts and it had more than 1600 commits behind `develop`, the conflict was resolved together with rebasing onto the `develop`. And the visual result of the **Issues to resolve table** is different from before, the new one is below.

![2022-05-24 18 17 33](https://user-images.githubusercontent.com/17420811/170012562-7ce78f81-ec22-4ff2-8679-f5ffa3f0b3eb.png)

puntope commented:
> Since there are code conflicts and it had more than 1600 commits behind develop, the conflict was resolved together with rebasing onto the develop. And the visual result of the Issues to resolve table is different from before, the new one is below.

Right now we use the issue details (if any) to populate the action. Maybe we could add a bit more of information in action field instead. Or just remove it. (If action field is null, the link goes directly to the docs without modal)

eason9487 committed:
Remove the `action` field overriding from the `merchant_quality_low` account issue to fit with the new UI of GMC account review request.

Address https://github.com/woocommerce/google-listings-and-ads/pull/1325#issuecomment-1135936945

eason9487 commented:
@puntope, thanks for the nice suggestions! 💯 

I adopted the second one in https://github.com/woocommerce/google-listings-and-ads/pull/1325/commits/58ceb42f752941a1facb3bea6d992669fa192716 since the `action` returned from Google API is `null` already. The latest screenshot is below and is also added to PR description.

![2022-05-25 19 06 54](https://user-images.githubusercontent.com/17420811/170249302-ff404cbd-b74d-4f28-825a-520638c7ef71.png)

eason9487 commented:
Removed the `status: blocked` label since the Google side has launched the Free and Enhanced Listings merge.
